internal/output/envvar.go

```diff
@@ -34,8 +35,12 @@ func NewEnvVar() *EnvVar {
 // Output Satisfies the Outputter interface and outputs AWS credentials as shell
 // export statements to STDOUT
 func (e *EnvVar) Output(c *config.Config, ac *aws.Credential) error {
-	fmt.Printf("export AWS_ACCESS_KEY_ID=%s\n", ac.AccessKeyID)
-	fmt.Printf("export AWS_SECRET_ACCESS_KEY=%s\n", ac.SecretAccessKey)
-	fmt.Printf("export AWS_SESSION_TOKEN=%s\n", ac.SessionToken)
+	export := "export"
+	if runtime.GOOS == "windows" {
+		export = "setx"
+	}
+	fmt.Printf("%s AWS_ACCESS_KEY_ID=%s\n", export, ac.AccessKeyID)
```

Comment:
Would it make sense to indicate users they have to run these commands manually?

Reply:
I added some additional clarification to the README